controls/radnavigationview/sdk-examples.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Developer Focused Examples
+page_title: Developer Focused Examples
+description: Check our &quot;Developer Focused Examples&quot; documentation article for the RadNavigationView {{ site.framework_name }} control.
```

Comment:
no need of the site.framework_name word and the if statements in the article because the control is WPF-only